pkg/release/build.go

```diff
@@ -290,6 +292,8 @@ const (
 	PlatformMacOS
 	// PlatformMacOSArm is the Darwin aarch6 target.
 	PlatformMacOSArm
+	// PlatformMacOSArmSigned is the Darwin aarch6 target.
+	PlatformMacOSArmSigned
```

Comment:
This is pretty confusing IMO. MacOSARMSigned and MacOSARM are not really different platforms at all. Instead maybe add a bool flag to `BuildOptions`? Or an extra bool argument to `SuffixFromPlatform`, given that it seems to be the only place where it matters.

Reply:
That seems to make sense, but whatever makes it most maintainable / understandable for you.